android/app/src/main/res/values-es/strings.xml

```diff
@@ -604,7 +604,7 @@
     <string name="routing_add_start_point">Busca o toca en el mapa para añadir el origen de la ruta</string>
     <!-- User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To". -->
     <string name="routing_add_finish_point">Busca o toca en el mapa para añadir el destino</string>
-    <string name="placepage_remove_stop">Eliminar</string>
+    <string name="placepage_remove_stop">Eliminar parada</string>
```

Comment:
Ok

Reply:
Please review in `strings.txt` and not in individual `.xml`-files